src/DependencyInjection/NeustaPimcoreHttpCacheExtension.php

```
@@ -20,20 +20,19 @@ public function loadInternal(array $mergedConfig, ContainerBuilder $container):
             $loader->load('object.php');
         }
 
         $container->getDefinition(StaticCacheTypeChecker::class)->setArgument('$types', [
-            ElementType::Asset->value => $mergedConfig['asset'],
-            ElementType::Object->value => $mergedConfig['object'],
-            ElementType::Document->value => $mergedConfig['document'],
-        ]);
+            ElementType::Asset->value => $mergedConfig['elements']['asset'],
+            ElementType::Object->value => $mergedConfig['elements']['object'],
+            ElementType::Document->value => $mergedConfig['elements']['document'],
     }
```

PHPStan (2.0.3)

Syntax error, unexpected '}', expecting ',' or ']' or ')' on line 37


(phpstan.parse)